daemon/container_operations_windows.go

```diff
@@ -58,3 +43,7 @@ func isLinkable(child *container.Container) bool {
 func enableIPOnPredefinedNetwork() bool {
 	return true
 }
+
+func (daemon *Daemon) supportsHotAdd() bool {
```

Comment:
The code prior to your changes infact had a compile time switch (between windows, unix, etc.)  But with this change, it is more dynamic. I dont have a personal preference 1 way or the other. If the current changes are simpler, I will just go with this approach.

Regarding the name can it be `isNetworkHotPluggable` ?